client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx

```diff
@@ -273,7 +274,8 @@ const InvestigationTable: React.FC = (): JSX.Element => {
                     )
                 }
                 else {
-                    return indexedRow[cellName as keyof typeof TableHeadersNames]
+                    return indexedRow[cellName as keyof typeof TableHeadersNames] ? 
```

Comment:
I think because this statement `indexedRow[cellName as keyof typeof TableHeadersNames]` is pretty complicated. you should place it on a variable so you don't write it twice.